### DISCUSSION TITLE
Add ChatGPT (Web)

### DISCUSSION THREAD

**Discussion by Enlight432** [immersive-translate/immersive-translate#1543] [Ideas]
### 新功能 | New Feature

I hope that talking about another open source extension is not against the rules

The ChatGPTBox extension has an option called ChatGPT (Web), which is enough to keep chat.openai.com open in a tab so that you can use its translation ability on different sites (that is, there is no need to api, etc.)

I wrote the name of this extension because I don't know how it does this

### 为什么您想要添加这些功能 | Why you want to add these features

Yes
Please add this item so that those who don't have an open ai subscription (Users who have a free subscription to OpenAI) can use gpt 3.5 for free in the direct translation of sites.

**Comment by theowenyoung** [comment]
Chatgpt Web's concurrent request limitations are so far beyond the needs of web page translation that it is almost impossible to support this feature

**Reply by Enlight432** [reply]
Is there a problem with Mouse Hover Section too?
